sdk/include/psdk/msctf.idl

```diff
@@ -46,10 +47,10 @@ cpp_quote("#define TF_E_STACKFULL            MAKE_HRESULT(SEVERITY_ERROR, FACILI
 cpp_quote("#define TF_E_SYNCHRONOUS          MAKE_HRESULT(SEVERITY_ERROR, FACILITY_ITF, 0x0208)")
 cpp_quote("#define TF_S_ASYNC                MAKE_HRESULT(SEVERITY_SUCCESS, FACILITY_ITF, 0x0300)")
 
-cpp_quote("EXTERN_C HRESULT WINAPI TF_InitSystem(VOID);")
-cpp_quote("EXTERN_C HRESULT WINAPI TF_UninitSystem(VOID);")
+cpp_quote("EXTERN_C BOOL WINAPI TF_InitSystem(VOID);")
+cpp_quote("EXTERN_C BOOL WINAPI TF_UninitSystem(VOID);")
```

Comment:
There are a couple of functions, in this file, that are _not_ exposed in the official msctf.idl from either Windows 2003, Windows 7 or Windows 10. These two are an example of that.

Reply:
`<msctf_undoc.h>`?

Reply:
That would be a good solution.

Reply:
Fixed.